homeassistant/components/climate/nest.py

```diff
@@ -115,6 +116,18 @@ def current_operation(self):
             return STATE_UNKNOWN
 
     @property
+    def current_hvac_state(self):
```

Comment:
Adding this property will not do anything. You will need to add a `device_state_attributes` method that returns it to get it be part of the state.